tests/pkg_micropython/Makefile

```diff
@@ -0,0 +1,29 @@
+# name of your application
+APPLICATION = pkg_micropython
+
+USEPKG += micropython
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
```

Comment:
I would suggest to add here (or somewhere it you think it's better) a list of unsupported boards, e.g 16bit and 8 bit boards.